### DISCUSSION TITLE
Submission 1 eval results, updates, more info

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas24_competition#42] [Announcements]
## Hello, hello

with a few hiccups along the way, we have evaluated (or at least tried to) all of the submissions for the first eval deadline. These are the results at the moment:
## First eval period results
| TeamID | Fruit count reported | Completion time | Path length | Return to start | Crashed / left area | Time limit |
|--------|----------------------|-----------------|-------------|-----------------|---------------------|------------|
| **01** | **21**               | **199.0**       | **176.3**   | **YES**         | **NO**              | **NO**     |
| 02     | N/A                  | N/A             | N/A         | NO              | YES                 | YES        |
| **03**     | **21**                   | **397.3**           | **204.3**       | **YES**             | **NO**               | **NO**         |
| 04     | N/A                  | N/A             | N/A         | NO              | NO                  | YES        |
| 05     | N/A                  | N/A             | N/A         | NO              | NO                  | YES        |
| 06     | 18                   | 134.0           | 166.64      | YES             | YES                 | NO         |
| **07**     | **18**               | **285.1**       | **225.6**   | **YES**         | **NO**              | **NO**     |
| 08     | 18                   | N/A             | N/A         | NO              | YES                 | YES        |
| 09     | 23                   | N/A             | N/A         | NO              | NO                  | YES        |
| 10     | 20                   | 104.1           | 171.8       | YES             | YES                 | NO         |
| 11     | N/A                  | N/A             | N/A         | NO              | NO                  | YES        |
| 12     | N/A                  | N/A             | N/A         | NO              | YES                 | YES        |
| 13     | 18                   | N/A             | N/A         | NO              | YES                 | YES        |
| 14     | N/A                  | N/A             | N/A         | N/A             | N/A                 | N/A        |
| 15     | N/A                  | N/A             | N/A         | NO             | NO                 | YES       |

Two bold results are clean runs. ~~**True value of eggplant fruit was 19** (although many reported 18 so I will double check)~~**True value of eggplant fruit was 18**. Make sure to check that the UAV does not leave the flying area. As already mentioned somewhere, the flying area is 3 meters around plant beds. In the collision check we include the propellers **(as far the collision check is concerned, the UAV is a box 105 cm wide/long)**. Time limit was 10 minutes. The mission was "Eggplant 1 6 13 15 17 19 25".

## More info
More info, including the run logs and team IDs will be sent to team leaders via email. 

## General guidelines for everyone

- Try not to rename docker images and containers, while not game-breaking it makes it easier for us to evaluate things, and you will probably get results faster.
- Make sure to make all scripts and nodes executable, also not game-breaking but it it makes it easier for us to evaluate things, and you will probably get results faster.
- Make sure your docker setup installs everything needed. We will not try and guess which package has which module and which version has the missing function. 
- Keep the structure of the submission package as is (we launch from startup/challenge folder). Also not game breaking, but helps a lot. 
- Make sure docker_build script defaults to the type of container you use (we will not guess whether you want gpu or not anymore). Also, we may skip some obscure instructions.txt file so please, make the default setup in the build and run scripts. 
- Make sure your zip is extractable.
- Make sure to fly within the specified flying area (to emphasize, the flying area is 3 meters around plant beds, in the collision check we include the propellers **(as far the collision check is concerned, the UAV is a box 105 cm wide/long)**), and make sure the UAV returns to the starting point. Return to the starting point means hovering (or if you are adventurous landing), but does not include flying through the starting point on the course to infinity or the nearest obstacle :).
- We will not consider late submissions, even if late by a couple of minutes.
- Deadline of 11:59 means 11:59 am, deadline of 23:59 means 11:59 pm.

## Next deadline
February 16th, 11:59 AOE.

## Time limit
For now, time limit stays the same (10 minutes). It may be lower for the final submission.

**Comment by Ro0412** [comment]
how do we know the team id?

**Reply by fpetric** [reply]
It should have been sent to team leaders of the teams that submitted. If anyone did not receive it, let me know

**Reply by Ro0412** [reply]
we didn't receive our id

**Reply by fpetric** [reply]
Can you send me an email?
